Requests/Builders/CatalogBuilders/ChangeCatalogItemsRequestBuilder.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace PyrusApiClient.Builders
+{
+    public class ChangeCatalogItemsRequestBuilder
+    {
+        private readonly PartialUpdateCatalogRequest _request;
+        public int CatalogId { get; }
+
+        public ChangeCatalogItemsRequestBuilder(int catalogId)
+        {
+            _request = new PartialUpdateCatalogRequest();
+            CatalogId = catalogId;
+        }
+
+        public ChangeCatalogItemsRequestBuilder(PartialUpdateCatalogRequest request, int catalogId)
+        {
+            _request = request;
+            CatalogId = catalogId;
+        }
+
+        public static implicit operator PartialUpdateCatalogRequest(ChangeCatalogItemsRequestBuilder builder)
+        {
+            return builder._request;
+        }
+
+        public ChangeCatalogItemsRequestBuilder UpsertItem(CatalogItem item)
+        {
+            _request.UpsertItems.Add(item);
```

Comment:
If it doesn't make sense to add a null item into the list then null check may be useful